### PR TITLE
Feature/16 leaflet v2

### PR DESCRIPTION
**Pull request by benjamineac**

### CODE REVIEW

**Review by chrisbc — APPROVED** [review]
@benjamineac thanks!

File: tsconfig.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
     "outDir": "./dist",
     "moduleResolution": "node"
   },
-  "include": ["src/**/*", "src/setupTests.ts"],
+  "include": ["src/**/*", "src/setupTests.ts", "decs.d.ts", "decs.d.ts"],
```
Comment (0qy) [inline tsconfig.json:12]: Duplicated 'decs.d.ts'

File: package.json [hunk 2]
```diff
@@ -54,33 +55,39 @@
...
     "@mui/material": "^5.5.2",
+    "@types/react-leaflet-fullscreen-plugin": "^1.0.0",
     "@visx/clip-path": "^2.1.0",
     "@visx/group": "^2.1.0",
     "@visx/legend": "^2.2.2",
     "@visx/responsive": "^2.8.0",
     "@visx/scale": "^2.2.2",
     "@visx/xychart": "^2.9.0",
+    "geojson": "^0.5.0",
+    "leaflet": "^1.8.0",
```
Comment (0qy) [inline package.json:86]: Add new dependencies to the list of 'peer dependencies'

Comment (0qy) [inline package.json:86]: Add new dependencies to the list of 'peer dependencies'

File: src/charts/LeafletMap.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
+import { GeoJsonObject } from 'geojson';
+import 'leaflet/dist/leaflet.css';
+import { LeafletMapProps } from '../interfaces/LeafletMap';
+import { MapContainer, TileLayer, GeoJSON } from 'react-leaflet';
+import Fullscreen from 'react-leaflet-fullscreen-plugin';
+
+const myStyle = {
```
Comment (0qy) [inline src/charts/LeafletMap.tsx:9]: Rename 'myStyle' to something more specific like 'geoJsonStyle' or something else that is clearer?

**Review by 0qy — COMMENTED** [review]
Also have you tried the component in the new playground-ts?

**Pull request merged**